docs/multitenancy.md

```diff
@@ -10,11 +10,13 @@ The following syntax rules apply to service:
 
 * Only alphanumeric characters and underscores (`_`) are allowed.
 * Maximum length is 50 characters.
+* Minimum length is 1 character.
 * If uppercase characters are used, they are converted to lowercase.
 
 The following syntax rules apply to subservice:
 
 * It must start with the `/` character.
 * Maximum length is 50 characters.
+* Minimum length is 1 character.
```

Comment:
This subtle detail belong to Orion doc. From an IoTP point of view, multipath sub services doesn't exist, so the current definition is correct.